pyscriptjs/environment.yml

```diff
@@ -1,11 +1,11 @@
 channels:
-  - defaults
-  - conda-forge
+-   defaults
```

Comment:
why this formatting change?

Reply:
It's from the auto-formatter. This one aligns list items with the parent. Indentation of the text is configurable. If eventually there is consensus that we'd prefer it differently, it wouldn't be tough to fork or contribute a new config option to the parent hook.